clients/go-ethereum/Dockerfile

```diff
@@ -17,6 +17,32 @@ FROM ethereum/client-go:$branch as builder
 
 ## ----
 
+## ----
+## Uncomment the steps below (and comment out the steps above!) to build
+## go-ethereum from a git branch.
+
+# FROM alpine:latest as builder
+# ARG user=ethereum
+# ARG repo=go-ethereum
+# ARG branch=master
+# RUN \
+#   apk add --update bash curl jq go git make gcc musl-dev         \
+#         ca-certificates linux-headers                         && \
+#   git clone --depth 1 --branch $branch                           \
+#       https://github.com/$user/$repo                          && \
+#   (cd go-ethereum && make geth)                               && \
+#   (cd go-ethereum                                             && \
+#   echo "{}"                                                      \
+#   | jq ".+ {\"repo\":\"$(git config --get remote.origin.url)\"}" \
+#   | jq ".+ {\"branch\":\"$(git rev-parse --abbrev-ref HEAD)\"}"  \
+#   | jq ".+ {\"commit\":\"$(git rev-parse HEAD)\"}"               \
+#   > /version.json)                                            && \
+#   cp go-ethereum/build/bin/geth /usr/local/bin/geth           && \
+#   apk del go git make gcc musl-dev linux-headers              && \
+#   rm -rf /go-ethereum && rm -rf /var/cache/apk/*
+
+## ----
```

Comment:
Wondering why you are adding these lines, since we already have the equivalent steps just above?

Reply:
This one makes it just a tiny bit easier to build from a git branch, instead of having to manually clone inside the folder, and having to checkout to switch branches to compile from a different source.

I've been going to the commit where this was removed and copying this excerpt to do my local testing, I just find it easier to test locally with this code.

Reply:
> since we already have the equivalent steps just above?

Yeah, no the steps above execute within the `ethereum/client-go` container as a builder, and is not suitable if you want to clone some non-official repository. LGTM